static_src/actions/notification_actions.js

```diff
@@ -0,0 +1,29 @@
+
+/*
```

Comment:
For this UI, I don't think notification actions and another notification store is required.

We used a separate error store and actions for global errors, relating to the whole application, so one store could decide how to render them. For the notifications here, they're all local to just the user invite section. This means the data for the notification should be stored in the UserStore, not a new notification store.

With this design as it stands, it's currently not separating the notifications in any way. So if the user invite adds a notification and then the app page adds a notification, there's no way of knowing that the one notification belongs on the user page and another belongs on the app page. Separating them would require adding more complex functionality to the notification store for each notification and where in the UI it would belong. Rather then creating a complex data structure like that, keeping a `inviteNotification` variable on the user store would be much simpler to update and access.

If we want global notifications, we should  probably refactor the error store/actions so you can specify the seriousness of the notification (really the error store just holds notifications for global errors occurring) rather then create a new store that's doing pretty much the same thing.